radixdlt/src/main/resources/schemas/atom.schema.json

```diff
@@ -334,6 +336,48 @@
         "nonce": { "type": "number" }
       },
       "description": "A particle which makes the containing Atom unique."
+    },
+    "registeredValidatorParticle": {
+      "type": "object",
+      "properties": {
+        "destinations": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/uid" },
+          "minItems": 1
+        },
+        "serializer": {
+          "type": "string",
+          "enum": [ "radix.particles.registered_validator" ]
+        },
+        "address": { "$ref": "#/definitions/address" },
+        "nonce": { "type": "number" }
+      },
+      "description": "A particle which marks a validator address as registered."
```

Comment:
I think we are missing a "required" tag here.  At least the "serializer" should be required, probably all the other things too.

Reply:
Yup, done.